code/modules/projectiles/projectile/bullets/shotgun.dm

```diff
@@ -112,3 +112,24 @@
 /obj/item/projectile/bullet/seed
 	damage = 4
 	stamina = 1
+
+//////////////////
+//    20 BORE   //
+//////////////////
+
+/obj/item/projectile/bullet/pellet/shotgun_buckshot/twentybore_buckshot
+	name = "20g buckshot pellet"
+	damage = 10
```

Comment:
If we assume that you give these the same pellets as buckshot (and since as of writing this you haven't so I couldn't say), then you may wish to be aware this is still 60 damage. Against a security officer that's reduced to 6 per pellet. 42 total. Compared to buckshot (12.5 to 8.75, 52.5 total) that's still not at all bad for hitting into seccies. It slows them at the very least.

Reply:
I think it should inherit whatever code that decides the 12g buckshot fires 6 pellets but instead just use a different pellet. I'll double check this. Going to wait on Kevin first though in case he decides to close it.
Vests have 0.3 bullet armour so it'll be 7 by default, the jumpsuit only has melee protection (I assume you meant 7 anyway since you said 42 total).
The idea behind it is that because the firing action is faster with the shotgun (fire + tap Z, load a new shell vs z, fire, z, load, z, fire, z, load, z fire) and you get a more diverse range of ammo, the shotgun should do a bit less damage. Making the shotguns use 20G and doing less damage is the only real way to make it not a straight upgrade to the rifle.

Reply:
Don't have it inheriting from 12g. Have it inherit from a parent shotgun shell unless you want to have guns using the parent ammo using these as well. Would it be realistic for the double-barrel to fit these new shells, for example, or for a bulldog magazine to fit these?

Reply:
The caliber in the ammo casing decides what you can load into a magazine. If I try to load casings with "shotgun20bore" it _should_ reject it from Bulldogs and Double-Barrels.
I'll make sure to test it all, I think I can make it a child/subtype of 12G and change the calibre, so there's no issues of it being loaded into the wrong guns.
In terms of double-barrel shotguns or bulldogs fitting 20G:
You'd need a 12G round that essentially has another bullet in it to so there's no chances of causing misfires. A misfire will probably just break the inside of the gun and make it inoperable without taking it apart and doing a bunch of repairs.
You could, if you wanted to, make either the double-barrel or bulldog only fire 20G. It's probably be a way to nerf the Bulldog.

Just a note on that though, there's a limited variety of 20G rounds you can get, At the time I just wanted to limit what you could have because, well, its improvised. In retrospect I probably should've made more 20G rounds.